### DISCUSSION TITLE
The message will send to public

### DISCUSSION THREAD

**Discussion by lauw0129** [NiekNijland/CSGO-Translator#9] [General]
The translated message will send to the public and everyone see it. Can I turn off this?

**Comment by NiekNijland** [comment]
@lauw0129 , Translations of messages sent in the all-chat and that are not written in your set language will be translated and sent to your team-chat so your team can see the translation as well. This is a feature in V3 beta 1.  Options to change is will come later. If you do not want this, remove the "-netconport" launch option or download V2.

![image](https://user-images.githubusercontent.com/43474728/115144298-3e699e80-a04c-11eb-8704-b5b481b990c2.png)
